execution_workflows/PAQR/PAQR_snakemake/workflow/rules/paqr.smk

```diff
@@ -0,0 +1,25 @@
+import os
+from snakemake.remote.HTTP import RemoteProvider as HTTPRemoteProvider
+from snakemake.utils import min_version
+min_version("6.0")
+
+
+# Download scripts required in the PAQR module
+HTTP = HTTPRemoteProvider()
+
+
+rule get_PAQR_scripts:
+    input:
+        HTTP.remote("raw.githubusercontent.com/zavolanlab/PAQR2/bd9781f1420ba39648120a6898038f49728541f3/scripts/{script}", keep_local=True)
+    output:
+        os.path.join(config["scriptsdir"], "{script}")
+    shell:
+        "mv {input} {output}"
+
+
+# Import the PAQR module
+module paqr:
+    snakefile: github("zavolanlab/PAQR2", path="Snakefile", commit="fdf56bc")
```

Comment:
I updated the conda env file in this PR, did you see that [here](https://github.com/iRNA-COSI/APAeval/blob/4d15f17b85ee18af4ffb0cfed5b42a5e3537a92e/apaeval_env.yaml). Could you double check whether you have snakemake 10? Maybe this will already resolve the issue.

Reply:
You're right, I missed that. I updated the conda environment by removing and reinstalling the environment and now have snakemake version 6.10.0.
Now I don't get this message anymore.